index.html

```diff
@@ -2074,61 +2075,61 @@
                         <table id="cubeIcons" alt="cubeIcons" label="cubeIcons" class="cubeIcon">
                             <tr>
                                 <td>
-                                    <img src="Pictures/Cube1.png" loading="lazy">
+                                    <img src="Pictures/Cube1.png" alt="" loading="lazy">
```

Comment:
can you add an actual label for alt?

Reply:
Don't know if I should answer to it. But alt is empty because image here used is purely cosmetic. And for screen readers informaion of importance is right bellow with cube tribute text